src/components/avatarHeader/avatarHeaderView.js

```diff
@@ -1,63 +1,48 @@
 import React from 'react';
-import { View, Text, SafeAreaView, TouchableOpacity } from 'react-native';
-import { useNavigation } from '@react-navigation/native';
-import LinearGradient from 'react-native-linear-gradient';
+import { View, Text, TouchableOpacity } from 'react-native';
 
 import { UserAvatar } from '../userAvatar';
 import { IconButton } from '../iconButton';
 
 // Styles
 import styles from './avatarHeaderStyles';
 
-const AvatarHeader =
-  ({ username, name, reputation, avatarUrl, showImageUploadActions, isUploading }) =>
-  () => {
-    const navigation = useNavigation();
+const AvatarHeader = ({
```

Comment:
if `AvatarHeader` is not being used anywhere else perhaps we can rename component to `EditAvatar`